rain_api_core/aws_util.py

```diff
@@ -149,52 +149,56 @@ def get_yaml_file(bucket, key):
     except ClientError as e:
         # The specified file did not exist
         log.error("Could not download yaml @ s3://{0}/{1}: {2}".format(bucket, key, e))
-        sys.exit()
+        return {}
```

Comment:
maybe raise?

Reply:
In that case the whole try-except can be removed since `get_yaml` already has similar code to log errors. No point in logging the same error twice